syntax = "proto3";

import "dronecore.proto";

package dronecorerpc;

service TelemetryRPC {
    rpc TelemetryPositionSubscription(UUID) returns(stream TelemetryPosition) {}
}

message TelemetryPosition {
    double latitude_deg = 1;
    double longitude_deg = 2;
    float absolute_altitude_m = 3;
    float relative_altitude_m = 4;
}
